setup.py

```diff
@@ -0,0 +1,32 @@
+import os
+import setuptools
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+deps = os.popen('pip freeze').readlines()
+# readlines() doesn't strip trailing newlines
+deps = [d.strip() for d in deps]
+
+setuptools.setup(
+    name="net-analysis",
+    version="0.0.1",
+    author="Jigsaw Operations, LLC",
+    author_email="fortuna@google.com",
+    description="Network censorship detection tools",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Jigsaw-Code/net-analysis",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "License :: OSI Approved :: Apache Software License",
```

Comment:
This doesn't seem to set the license. I get:
```
$ pip show net-analysis
Name: net-analysis
Version: 0.0.1
Summary: Network censorship detection tools
Home-page: https://github.com/Jigsaw-Code/net-analysis
Author: Jigsaw Operations, LLC
Author-email: fortuna@google.com
License: UNKNOWN
Location: /usr/local/google/home/fortuna/.local/share/virtualenvs/piptest-i3ZKsPr0/lib/python3.7/site-packages
Requires: 
Required-by:
```

See the "license" field missing.

Reply:
I'm not sure what the issue is, as I followed the directions for setup.py which indicate to use the classifiers and a LICENSE file with no other mention of licenses.  I'll keep looking